distributions/jamulus-server.desktop.in

```diff
@@ -0,0 +1,19 @@
+[Desktop Entry]
+Name=Jamulus (server)
+Name[fr]=Jamulus (serveur)
```

Comment:
For German, you can add:
```
Name[de]=Jamulus (Server)
```

(Maybe the proper translation for `server` for other languages can be stolen from `.ts` files?)

Reply:
> As said on other channels, I'm not the Debian Packager of the project and I don't have the necessary knowledge to make those changes and update that sort of things. When merged, we will have to create an issue and tell the Debian packager that this is available.

But I guess you intend to use it from Arch Linux, don't you? Isn't `Jamulus.pro` used there as well? I had assumed so because it is not distribution-specific but rather the platform-agnostic build logic.

A `jamulus-server.desktop.in` which (obviously) requires dynamic replacements does not sound too helpful if those replacements are not managed the same as for other .in files.

> A translation is necessary only if it is different in the translated language.

Yes. I added the suggestion for German because of different case.

Reply:
>
> * `jamulus-server.desktop.in` should probably be added to `DISTFILES` and `QMAKE_SUBSTITUES`. The resulting `jamulus-server.desktop` should probably be added to `dekstop.files` in `Jamulus.pro`
> 

I don't know what is `QMAKE_SUBSTITUES`

Reply:
> Yes, but PKGBUILD of Archlinux is doing all the good things by itself.

As far as I see, it still runs `qmake` and `make`, so it would be affected by any changes here (or would lack the changes if we skip the `Jamulus.pro` part).

> I don't know what is QMAKE_SUBSTITUES

Sorry, I had a typo there -- it's QMAKE_SUBSTITUTES and is pat of `Jamulus.pro`.

I'm still a bit unsure about the scope of this PR. What should be the goal?

1. Should the PR be limited to adding the new icon files? I'd be OK with that, but we should probably not add `jamulus-server.desktop.in` then as it will not be working, as far as I understand.
2. Should the PR introduce a Jamulus Server desktop icon and add the new SVG-based logo versions? Then I think we should do it properly. Doing it properly would mean, as far as I understand:
    - Ensure that `jamulus-server.desktop.in` is properly converted to `jamulus-server.desktop` similar to `jamulus.desktop.in` (I assume that `QMAKE_SUBSTITUTES` does that) and marked/installed correctly (`desktop.files`?)
    - Check if and how icon references need to be updated (`icon.files`, `DISTFILES`). Ensure that SVG is supported the same as the existing PNG.

Note: I don't know any qmake specifics either. I'm just working by tracing where the existing files (`jamulus.desktop.in`, `jamulus.png`) are being referenced and I'm checking whether it sounds logical to update that for the new files as well.

Personally, I think option 2 should be aimed for.

I'm happy to be convinced otherwise, btw, maybe I'm missing something why even the partial changes would be helpful or could work. Right now my understand is that it will neither directly benefit official Jamulus builds nor downstream distribution builds (e.g. Arch Linux' PKGBUILD seems to rely on the *built* `jamulus.desktop` file, so I'd assume that just adding `jamulus.desktop.in` will not be sufficient).

Reply:
You are right.
Learning deeper and I will add a commit soon with all the necessary stuff.
For now, I tested after reading the freedesktop documentation : [https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-0.6.html](Icons)
It's ok with svg icons.

Reply:
Ok, done and tested here.
We are reaching the same level as Windows and macOS desktop installation.